doc: nrf: Inform about dfu_application.zip update (#15958)
nrfconnect/sdk-nrf: scripts/hid_configurator/README.rst

The configuration channel uses a cross-platform HIDAPI library to communicate with HID devices attached to the operating system.
See `Dependencies`_ for more information.

.. note::

>    Since NCS 2.7 release, format version of the :file:`dfu_application.zip` file generated by the build system was updated from ``0`` to ``1``.

Reviewer: ```suggestion
   Since the |NCS| v2.7.0 release, the format version of the :file:`dfu_application.zip` file generated by the build system was updated from ``0`` to ``1``.
```

>    The format version ``1`` support was also added to the script in NCS 2.7 release.

Reviewer: ```suggestion
   The format version ``1`` support was also added to the script in the |NCS| v2.7.0 release.
```

>    Make sure to use updated version of the script for archives in the new format version.

Reviewer: ```suggestion
   Make sure to use the updated version of the script for archives in the new format version.
```

   The new version of the script can also handle archives in legacy format version.

>    If updating script on given host is not possible, you can manually align content of a zip archive generated with format version ``1`` to format version ``0``.

Reviewer: ```suggestion
   If you cannot update the script on a given host, you can manually align the content of the zip archive generated with format version ``1`` to format version ``0``.
```
